cypress/integration/plugins/dashboards-assistant/mds_conversation_history_spec.js

```diff
@@ -9,8 +9,18 @@ if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
   describe('Assistant conversation history spec', () => {
     let restoreShowHome;
     let restoreNewThemeModal;
+    let dataSourceId;
 
     before(() => {
+      if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
```

Comment:
Can we reuse the `setDefaultDataSourceForAssistant`? Maybe we can return the created data source id in `setDefaultDataSourceForAssistant` result.

Reply:
Sure, the code had been updated.